docs/arm64e-Deployment.md

```diff
@@ -26,14 +26,18 @@ There are two ways to overcome this:
 
   Once downloaded, opening the .xip file will begin extracting it. After extraction, rename the app to not conflict with your primary installation of Xcode. A good choice is `Xcode-11.7.app`.
 
-  Finally, change your selected Xcode command line tools version by using the following command:
+  Finally, change your selected Xcode command line tools version:
 
   ```bash
-  sudo xcode-select -switch /Applications/Xcode-11.7.app
+  sudo xcode-select -switch /Applications/Xcode-11.7.app/Contents/Developer
```

Comment:
Strictly it does make more sense to pass the Developer dir, but xcode-select figures it out if you just pass Xcode.app. Slightly easier to remember/less typing. Just a suggestion, will approve this anyway and you can change it before merging if you prefer.

Reply:
hmm I'd still prefer that we suggest passing the developer dir for consistency, since that's what everything else uses and is also what `xcode-select -p` reports.